oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -550,6 +550,8 @@ public NodeDocument call() throws Exception {
                 return findUncached(collection, key, docReadPref);
             } catch (MongoException e) {
                 ex = e;
+                LOG.error("Read Fails with an exception" + e);
+                e.printStackTrace();
```

Comment:
```suggestion
                LOG.error("findUncachedWithRetry : read fails with an exception" + e, e);
```

Reply:
would adding the exception stacktrace to the log make sense or is that too verbose? (e.printStackTrace() otherwise just prints to standard out)

Reply:
~~~
LOG.error("findUncachedWithRetry: read fails with an exception: " + e, e);
~~~